dynatrace-operator-google-marketplace/schema.yaml

```diff
@@ -139,6 +139,83 @@ properties:
     title: "Automatically update OneAgent"
     default: true
 
+  codeModules.enabled:
+    type: boolean
+    description: "Enable code modules"
+    title: "Enable code modules"
+  codeModules.volume:
+    type: string
+    description: "Define a volume the OneAgent binary will be taken from"
+    title: "CodeModules volume"
+  codeModules.resources:
+    type: string
+    description: "Resource settings for OneAgent container"
+    title: "CodeModules resources"
+  codeModules.selector:
+    type: string
+    description: "Set the label selector to define which namespaces to inject into"
+    title: "CodeModules selector"
+
+  infraMonitoring.enabled:
+    type: boolean
+    description: "Enable infrastructure monitoring"
+    title: "Enable infrastructure monitoring"
+  infraMonitoring.nodeSelector:
+    type: string
+    description: "Set node selector to control on which nodes the OneAgent will be deployed"
+    title: "Infra node selector"
+  infraMonitoring.tolerations:
+    type: string
+    description: "Tolerations to include with the OneAgent DaemonSet"
+    title: "Infra tolerations"
+  infraMonitoring.waitReadySeconds:
```

Comment:
AFAIK we do not have the waitReadySeconds anymore - please double check in the Operator repo for classicFullStack

Reply:
removed, also removed from classicFullStack because it is not used anymore there either